app.yaml

```diff
@@ -5,6 +5,12 @@ api_version: 1
 threadsafe: true
 
 handlers:
+- url: /wavedroplet.css
```

Comment:
at some point it becomes a good idea to just create a static/ subdir and export all the files in there with a single rule.  This way is fine but is more work.